plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -219,6 +219,37 @@ func (e *Elasticsearch) createHttpClient() (*http.Client, error) {
 	return client, nil
 }
 
+func (e *Elasticsearch) generateIndividualStatsUrlPart() string {
+	if len(e.IndividualStats) == 0 {
+		return ""
+	}
+
+	return "/" + strings.Join(e.IndividualStats, ",")
+}
+
+func (e *Elasticsearch) generateStatsUrl(baseUrl string) string {
+	var url string
+
+	individualStatsUrlPart := e.generateIndividualStatsUrlPart()
+
+	if e.Local {
+		url = baseUrl + statsPathLocal
+	} else {
+		url = baseUrl + statsPath
+	}
+
+	if individualStatsUrlPart == "" {
+		return url
+	}
+
+	return url + individualStatsUrlPart
+}
```

Comment:
I think this can be simplified to:
```go
+func (e *Elasticsearch) generateStatsUrl(baseUrl string) string {
	var url string

	if e.Local {
		url = baseUrl + statsPathLocal
	} else {
		url = baseUrl + statsPath
	}

	return url + e.generateIndividualStatsUrlPart()
}
```

Reply:
Yes!